save.js

```diff
@@ -3,9 +3,10 @@
   chrome.storage.local.get("scroll-mark", data => {
     const scrollMarkData = data["scroll-mark"];
     const offset = window.pageYOffset;
+    const total = document.body.clientHeight;
     const newData = scrollMarkData
-      ? { ...scrollMarkData, [url]: offset }
-      : { [url]: offset };
+      ? { ...scrollMarkData, [url]: { offset, total } }
```

Comment:
One last thing can you also include the title of the page in this object using `document.title`.